packages/devtools/web/styles.css

```diff
@@ -762,10 +764,8 @@ html [full] {
     opacity: 0.5;
 }
 
-.action-button img {
-    width: 18px;
-    height: 18px;
-    vertical-align: middle;
+.action-button .flutter-icon {
```

Comment:
is it intentional that you have deleted the
.action-button img
case?

Reply:
yeah, we no longer have case.